applications/machine_learning/configuration/thingy52_nrf52832/app_ZDebug.conf

```diff
@@ -13,6 +13,7 @@ CONFIG_ML_APP_EI_DATA_FORWARDER_SENSOR_EVENT_DESCR="accel_xyz"
 CONFIG_ML_APP_LED_STATE=y
 CONFIG_ML_APP_LED_STATE_VALUE_THRESH=700
 CONFIG_ML_APP_LED_STATE_ANOMALY_THRESH=300
+CONFIG_ML_APP_LED_STATE_PREDICTION_STREAK_THRESH=3
```

Comment:
I think this should not e a part of this commit, could you split this into two?

Reply:
done